src/common/engine.js

```diff
@@ -279,7 +286,7 @@ class CurrencyEngine {
         if (!this.txIdMap[currencyCode]) {
           this.txIdMap[currencyCode] = {}
         }
-        this.txIdMap[currencyCode][tx.txid] = i
+        this.txIdMap[currencyCode][normalizeAddress(tx.txid)] = i
```

Comment:
This isn't an address. Maybe the helper should be called 'normalizeHex' instead.